docs/manual/manual_telemetry.rst

```diff
@@ -0,0 +1,235 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_telemetry:
+
+***********************************************
+ESC Telemetry
+***********************************************
+Flight controllers, including flight controllers running `PX4 <https://px4.io/>`_ or `Ardupilot <https://ardupilot.org/>`_ firmware, often support receiving ESC telemetry messages from connected ESCs. These messages provide the flight controller with details
+about the state of the ESC and the motor or motors it is controlling. Vertiq modules support a range of these telemetry messages across different protocols. This section provides
+details on the structure of the telemetry messages supported by Vertiq modules and how to use them.
+
+.. _telemetry_module_support:
+
+Module Support
+===============
+
+Speed Modules
+**************
+
+.. table:: Speed Module Support for Telemetry
+
+	+-------------+------------------------------------+---------------------------------------------------------------------------------------------------------------+

```

Comment:
could we possibly make this a "tight table" that will keep it from needing the scroll bar. I did this on the GPIO stuff: 
`.. table:: GPIO Registers
		:widths: 16 20 20 20 20 20 20 20 20 6 50
		:class: tight-table

		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
		| Register Name | Bit 7            | Bit 6            | Bit 5            | Bit 4            | Bit 3            | Bit 2            | Bit 1            | Bit 0            | Access | Description                                                                                                                                                                       |
		+===============+==================+==================+==================+==================+==================+==================+==================+==================+========+===================================================================================================================================================================================+
		| GPIO Mode     | GPIO 8 Mode      | GPIO 7 Mode      | GPIO 6 Mode      | GPIO 5 Mode      | GPIO 4 Mode      | GPIO 3 Mode      | GPIO 2 Mode      | GPIO 1 Mode      | R/W/S  | Sets the GPIO pin to a specified mode, input or output. 0 indicates input, 1 indicates output                                                                                     |
		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
		| Input Values  | GPIO 8 Input     | GPIO 7 Input     | GPIO 6 Input     | GPIO 5 Input     | GPIO 4 Input     | GPIO 3 Input     | GPIO 2 Input     | GPIO1 Input      | RO     | A read only value that indicates the value of each input pin when set to input mode                                                                                               |
		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
		| Output Values | GPIO 8 Ouput     | GPIO 7 Ouput     | GPIO 6 Ouput     | GPIO 5 Ouput     | GPIO 4 Ouput     | GPIO 3 Ouput     | GPIO 2 Ouput     | GPIO 1 Ouput     | R/W/S  | The value to send/being sent on an output GPIO line                                                                                                                               |
		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
		| Use Pull      | GPIO 8 Use Pull  | GPIO 7 Use Pull  | GPIO 6 Use Pull  | GPIO 5 Use Pull  | GPIO 4 Use Pull  | GPIO 3 Use Pull  | GPIO 2 Use Pull  | GPIO 1 Use Pull  | R/W/S  | Determines whether or not to use the internal pull-up or pull-down resistor on input mode. If 0, use no pull, if 1 use the pull type specified by the corresponding Pull Type bit |
		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
		| Pull Type     | GPIO 8 Pull Type | GPIO 7 Pull Type | GPIO 6 Pull Type | GPIO 5 Pull Type | GPIO 4 Pull Type | GPIO 3 Pull Type | GPIO 2 Pull Type | GPIO 1 Pull Type | R/W/S  | Determines the type to use on an input pin, if Use Pull is set high. 0 indicates pull down, 1 indicates pull up.                                                                  |
		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
		| PP/OD         | GPIO 8 PP/OD     | GPIO 7 PP/OD     | GPIO 6 PP/OD     | GPIO 5 PP/OD     | GPIO 4 PP/OD     | GPIO 3 PP/OD     | GPIO 2 PP/OD     | GPIO 1 PP/OD     | R/W/S  | Determines the output mode, push-pull or open-drain. 0 indicates push-pull, 1 indicates open-drain                                                                                |
		+---------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+`